### ISSUE TITLE
Composer: but these conflict with your requirements or minimum-stability.

### ISSUE THREAD

**Issue by ghost** [laravel-notification-channels/pushover#32]
Using Laravel 5.8 I get this when trying to install:

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - laravel-notification-channels/pushover 1.2.3 requires nesbot/carbon ^1.21 -> satisfiable by nesbot/carbon[1.21.0, 1.22.0, 1.22.1, 1.23.0, 1.24.0, 1.24.1, 1.24.2, 1.25.0, 1.25.1, 1.25.3, 1.26.0, 1.26.1, 1.26.2, 1.26.3, 1.26.4, 1.26.5, 1.26.6, 1.27.0, 1.28.0, 1.29.0, 1.29.1, 1.29.2, 1.30.0, 1.31.0, 1.31.1, 1.32.0, 1.33.0, 1.34.0, 1.34.1, 1.34.2, 1.34.3, 1.34.4, 1.35.0, 1.35.1, 1.36.0, 1.36.1, 1.36.2, 1.37.0, 1.37.1, 1.38.0, 1.38.1, 1.38.2, 1.38.3, 1.38.4] but these conflict with your requirements or minimum-stability.
    - laravel-notification-channels/pushover 1.2.2 requires nesbot/carbon ^1.21 -> satisfiable by nesbot/carbon[1.21.0, 1.22.0, 1.22.1, 1.23.0, 1.24.0, 1.24.1, 1.24.2, 1.25.0, 1.25.1, 1.25.3, 1.26.0, 1.26.1, 1.26.2, 1.26.3, 1.26.4, 1.26.5, 1.26.6, 1.27.0, 1.28.0, 1.29.0, 1.29.1, 1.29.2, 1.30.0, 1.31.0, 1.31.1, 1.32.0, 1.33.0, 1.34.0, 1.34.1, 1.34.2, 1.34.3, 1.34.4, 1.35.0, 1.35.1, 1.36.0, 1.36.1, 1.36.2, 1.37.0, 1.37.1, 1.38.0, 1.38.1, 1.38.2, 1.38.3, 1.38.4] but these conflict with your requirements or minimum-stability.
    - laravel-notification-channels/pushover 1.2.1 requires nesbot/carbon ^1.21 -> satisfiable by nesbot/carbon[1.21.0, 1.22.0, 1.22.1, 1.23.0, 1.24.0, 1.24.1, 1.24.2, 1.25.0, 1.25.1, 1.25.3, 1.26.0, 1.26.1, 1.26.2, 1.26.3, 1.26.4, 1.26.5, 1.26.6, 1.27.0, 1.28.0, 1.29.0, 1.29.1, 1.29.2, 1.30.0, 1.31.0, 1.31.1, 1.32.0, 1.33.0, 1.34.0, 1.34.1, 1.34.2, 1.34.3, 1.34.4, 1.35.0, 1.35.1, 1.36.0, 1.36.1, 1.36.2, 1.37.0, 1.37.1, 1.38.0, 1.38.1, 1.38.2, 1.38.3, 1.38.4] but these conflict with your requirements or minimum-stability.
    - laravel-notification-channels/pushover 1.2.0 requires nesbot/carbon ^1.21 -> satisfiable by nesbot/carbon[1.21.0, 1.22.0, 1.22.1, 1.23.0, 1.24.0, 1.24.1, 1.24.2, 1.25.0, 1.25.1, 1.25.3, 1.26.0, 1.26.1, 1.26.2, 1.26.3, 1.26.4, 1.26.5, 1.26.6, 1.27.0, 1.28.0, 1.29.0, 1.29.1, 1.29.2, 1.30.0, 1.31.0, 1.31.1, 1.32.0, 1.33.0, 1.34.0, 1.34.1, 1.34.2, 1.34.3, 1.34.4, 1.35.0, 1.35.1, 1.36.0, 1.36.1, 1.36.2, 1.37.0, 1.37.1, 1.38.0, 1.38.1, 1.38.2, 1.38.3, 1.38.4] but these conflict with your requirements or minimum-stability.
    - Installation request for laravel-notification-channels/pushover ^1.2 -> satisfiable by laravel-notification-channels/pushover[1.2.0, 1.2.1, 1.2.2, 1.2.3].


Installation failed, reverting ./composer.json to its original content.
```

**Comment by kapersoft** [comment]
The latest 2.0-release with support for Laravel 5.8 is not added to packagist (ping @casperboone).

As a workaround you can add the following to your composer.json:
```json
"repositories": [
  {
    "type": "git",
    "url": "https://github.com/laravel-notification-channels/pushover"
  }
],
```

**Comment by ghost** [comment]
Saw that, but the 2.0 release still lists an old version:

` "nesbot/carbon": "^1.21"`

**Comment by casperboone** [comment]
PR to fix it is appreciated :)

**Comment by kapersoft** [comment]
I've made a PR: https://github.com/laravel-notification-channels/pushover/pull/34

Can you also add the new release in packagist?

**Comment by casperboone** [comment]
There was an issue with the Carbon dependency. It's probably fixed in version 2.0.1. This version is not available on packagist yet (might take a while, I cannot manage the package in packagist 🎉, but all releases seem to become available after a while), but you can use the dev-master version.
